src/containers/Bird/Bird.tsx

```diff
@@ -0,0 +1,20 @@
+import styles from './Bird.module.scss';
+
+import classNames from 'classnames';
+import { IPropsBird } from './Bird.interface';
+import { P } from '@shared/PTextComponent/P';
+
+const Bird = ({ bird, onBirdNameClick, designType, stateStyleValue }: IPropsBird) => {
+  const classes = classNames(styles.default, {
+    [styles[designType]]: stateStyleValue,
+  });
+
+  return (
+    <li className={styles.birdValue} id={`${bird.id}`} onClick={() => onBirdNameClick(bird)}>
```

Comment:
Не прокидываем напрямую хендлеры. Выносим их в отдельную функцию

const handleClick  = () => onBirdNameClick(bird)

```suggestion
    <li className={styles.birdValue} id={`${bird.id}`} onClick={handleClick}>
```